src/core_modules/capture-core/components/WidgetFeedback/WidgetFeedbackContent/WidgetFeedbackContent.js

```diff
@@ -11,30 +11,40 @@ const styles = {
     },
     unorderedList: {
         paddingLeft: spacersNum.dp16,
-        marginTop: '0px',
-        lineHeight: '1.375',
-        fontSize: spacers.dp16,
+        marginTop: 0,
+        fontSize: '14px',
+        lineHeight: '19px',
         color: colors.grey900,
     },
     noFeedbackText: {
         color: colors.grey600,
-        paddingLeft: spacersNum.dp16,
         fontWeight: 400,
-        fontSize: spacers.dp16,
-        marginTop: '0px',
+        fontSize: '14px',
+        lineHeight: '19px',
+        marginTop: 0,
     },
     listItem: {
-        marginBottom: spacersNum.dp4,
+        marginBottom: spacersNum.dp8,
         '&::marker': {
             color: colors.grey500,
         },
+        '& + &': {
+            marginBottom: 0,
+        },
     },
 };
 
 
 const WidgetFeedbackContentComponent = ({ widgetData, emptyText, classes }: ContentType) => {
     if (!widgetData?.length) {
-        return <p data-test="widget-content" className={classes.noFeedbackText}>{emptyText}</p>;
+        return (
+            <div
+                data-test="widget-content"
+                className={classes.container}
```

Comment:
Using the `container` for layout styles, rather than styling the individual objects.